src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  let sale = 0;
+
+  if (days > SHORT_TERM && days < LONG_TERM) {
+    sale = 20;
+  }
+
+  if (days >= LONG_TERM) {
+    sale = 50;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions, and avoid using `else` after `if` with `return`. Since the second `if` condition is independent and overrides the first one when `days` is equal to or greater than `LONG_TERM`, consider returning the total cost directly from within the `if` blocks.